content/en/docs/collector/getting-started.md

```diff
@@ -99,6 +99,9 @@ OpenTelemetry Collector instance, with features such as automatic upgrade
 handling, `Service` configuration based on the OpenTelemetry configuration,
 automatic sidecar injection into deployments, among others.
 
+For a comprehensive guide to using the collector with Kubernetes, see
+[Kubernetes Getting Started](../../kubernetes/getting-started).
```

Comment:
In general, use relative page links for pages in the same section, not across sections.

```suggestion
[Kubernetes Getting Started](/docs/kubernetes/getting-started/).
```